### PR TITLE
WordPress (VIP) CodeSniffs to test suite / CI

### PR DESCRIPTION
**Pull request by renatonascalves**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------
It adds phpcs support to travis.

Does this close any currently open issues?
------------------------------------------
#627 

Any other comments?
-------------------
This is not ready yet, as I want to test this with docker.

### CODE REVIEW

**Comment by jasonbahl** [comment]
🎊🎉🎈🍾

File: wp-graphql.php [hunk 1]
```diff
@@ -280,11 +276,9 @@ public function setup_types() {
 			/**
 			 * Setup the settings, post_types and taxonomies to show_in_graphql
 			 */
-			\WPGraphQL::show_in_graphql();
-			\WPGraphQL::get_allowed_post_types();
-			\WPGraphQL::get_allowed_taxonomies();
-
-
+			self::show_in_graphql();
```
Comment (renatonascalves) [inline wp-graphql.php]: @jasonbahl I do wonder why it was calling itself that way. Any particular reason?

**Comment by renatonascalves** [comment]
Ok! Base implementation introduced! PHPCS is checking code against, right now, only `wp-graphql.php`, for WordPress Coding Standards.

There is a `phpcs.xml.dist` in place where we are saying which standards to follow and which rules or files to remove.

Also, for the time being, I'm saying that Travis can fail for the phpcs tests.

File: wp-graphql.php (hunk 1 above)
Comment (jasonbahl) [reply wp-graphql.php]: no reason that I can recall 🤔 

I think at one point they were in different classes or something. I don't recall. But this makes more sense!

**Comment by jasonbahl** [comment]
@renatonascalves Thanks for this. I think it's a great addition as we start moving to standardize the code style.

**Pull request merged**